rezervo/utils/santa_utils.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
+
+
+def check_santa_time() -> bool:
+    return datetime.now().month == 12
```

Comment:
Can't have santa too early or too late :smirk: 
```suggestion
from datetime import datetime

import pytz


def check_santa_time() -> bool:
    # TODO: clean this
    return datetime.now(pytz.timezone("Europe/Oslo")).month == 12

```